sqlmesh/core/dialect.py

```diff
@@ -645,3 +646,21 @@ def extract_columns_to_types(query: exp.Subqueryable) -> t.Dict[str, exp.DataTyp
         expression.output_name: expression.type or exp.DataType.build("unknown")
         for expression in query.selects
     }
+
+
+def find_tables(expression: exp.Expression, dialect: DialectType = None) -> t.Set[str]:
+    """Find all tables referenced in a query.
+
+    Args:
+        expressions: The query to find the tables in.
+        dialect: The dialect to use for normalization of table names.
+
+    Returns:
+        A Set of all the table names.
+    """
+    return {
+        normalize_model_name(table, dialect=dialect)
```

Comment:
Was there a bug here previously? How come we didn't normalize this table?

Reply:
yep, it was a bug